pages/archive/[page].js

```diff
@@ -18,7 +19,7 @@ export default function StoryArchive({
   return (
     <>
       <Navigation navigationData={navigationData} />
-      {renderRichText(archiveTitle)}
+      <H1 h1Data={archiveTitle} />
```

Comment:
If you go to `/archive` (no ID after) it sends to an error page. Is this something to be resolved in another PR?

Reply:
This is something we were going to resolve in another PR!